airesources/CSharp/Halite2/hlt/Networking.cs

```diff
@@ -53,8 +53,6 @@ public static void SendMoves(IEnumerable<Move> moves)
 
         private static String ReadLine()
```

Comment:
Can you check if this terminates properly when there is no input from halite? This behaves differently in Linux and Windows - in Linux it seems halite can kill the bot successfully but on Windows bot client library has to kill itself when stdin is closed.

Reply:
I suspect part of this try/catch originally was to exit cleanly, but I'm not sure.

Reply:
This is an example of what cpp lib has to do just for windows: https://github.com/HaliteChallenge/Halite-II/blob/99cec0c7763f6a74ff46d6c3a9d89d5c28638d40/airesources/C%2B%2B/hlt/hlt_in.hpp#L127